app/controllers/workers_controller.rb

```diff
@@ -2,12 +2,13 @@ class WorkersController < ApplicationController
   before_action :logged_in_user, only: %i[index show new create edit update]
   before_action :set_worker, only: %i[show edit update]
   rescue_from ActiveRecord::RecordNotFound, with: :render_not_found
-
+  attr_accessor :form_labels, :form_text_fields, :form_submit_button
+  
```

Comment:
@Timilehin08 Hi Timi! Great work! I wonder what is attr_accessor does? I didn't add it to my code. I wonder If I am missing something important. ☺️

Reply:
Hey @Timilehin08 , it looks like you're exposing bootstraps classes here but those aren't accessed by controller methods. I'd recommend removing it for this PR.

Reply:
@GabrielleEaston I've found this[ thread ](https://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby) helped to understand what attr_accessor does. 

TLDR: Ruby doesn't technically have the concept of class variables (even though controller files resemble classes defined in other languages). The attr_accessor method helps to expose variables defined in one class to all the other methods in that file.

Reply:
Thank you for reviewing! @ncbui @GabrielleEaston.
Oops! Okay, i will.

@GabrielleEaston 
I learnt attr_accessor is a method and class is not a definition in Ruby (it is "just a definition" in languages like Java), but it is an expression that evaluates. It is during this evaluation that the attr_accessor method is invoked which in turn modifies the variables in one method to all other methods in the file. still learning haha, so I definitely used it wrongly here.

Thank you again!, I'll make the changes @ncbui